auth/api/iam/types.go

```diff
@@ -34,6 +34,9 @@ type DIDDocumentMetadata = resolver.DocumentMetadata
 // VerifiableCredential is an alias
 type VerifiableCredential = vc.VerifiableCredential
 
+// VerifiableCredential is an alias
```

Comment:
wrong comment, since you're no longer referencing a VC from the OAS, the VerifiableCredential alias can be removed?

Reply:
It will be back after moving OpenID4VCI from the VCR, but will remove for now.